searx/utils.py

```diff
@@ -638,7 +638,28 @@ def _get_fasttext_model() -> "fasttext.FastText._FastText":
 
 
 def detect_language(text: str, threshold: float = 0.3, min_probability: float = 0.5) -> Optional[str]:
-    """https://fasttext.cc/docs/en/language-identification.html"""
+    """
+    The language detection is done by using the fastText_ library (`python
+    fasttext`_). fastText_ distributes the `language identification model`_, for
+    reference:
+    - `FastText.zip: Compressing text classification models`_
+    - `Bag of Tricks for Efficient Text Classification`_
+    The `language identification model`_ support the language codes (ISO-639-3)::
+    af als am an ar arz as ast av az azb ba bar bcl be bg bh bn bo bpy br bs bxr
+    ca cbk ce ceb ckb co cs cv cy da de diq dsb dty dv el eml en eo es et eu fa
+    fi fr frr fy ga gd gl gn gom gu gv he hi hif hr hsb ht hu hy ia id ie ilo io
+    is it ja jbo jv ka kk km kn ko krc ku kv kw ky la lb lez li lmo lo lrc lt lv
+    mai mg mhr min mk ml mn mr mrj ms mt mwl my myv mzn nah nap nds ne new nl nn
+    no oc or os pa pam pfl pl pms pnb ps pt qu rm ro ru rue sa sah sc scn sco sd
+    sh si sk sl so sq sr su sv sw ta te tg th tk tl tr tt tyv ug uk ur uz vec vep
+    vi vls vo wa war wuu xal xmf yi yo yue zh
+
+    .. _fastText: https://fasttext.cc/
+    .. _python fasttext: https://pypi.org/project/fasttext/
+    .. _language identification model: https://fasttext.cc/docs/en/language-identification.html
+    .. _Bag of Tricks for Efficient Text Classification: https://arxiv.org/abs/1607.01759
+    .. _`FastText.zip: Compressing text classification models`: https://arxiv.org/abs/1612.03651
+    """
```

Comment:
This is not the function description .. I miss a description of function's arguments .. for what e.g. is `min_probability`?

Reply:
> In your review of https://github.com/searxng/searxng/pull/2019 you added min_probability back, but it was not clear to me why .. did I have overseen or missed something?

You are right, `min_probability` there is no need. I removed it.

The docstring is updated too.

Note: simplified and traditional Chinese are both detected as `zh`.